src/app/globals.css

```diff
@@ -1,33 +1,66 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 50px;
+}
+
+p {
+  font-family: 'Markazi Text';
+  font-weight: 700;
+  font-size: 20px;
+  margin: 0px;
+}
+
+.page {
+  text-align: center;
+  display: flex;  
+  align-items: center;
+  flex-direction: column;
+}
+
+.desc {
+  padding-bottom: 88px;
+}
+
+.main {
+  display: flex;  
+  align-items: center;
+  width: 247px;
+  height: 337px;
+  background: #FABC3C;
+  justify-content: center;
+}
 
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+.main button {
+  width: 19px;
+  height: 24px;
+  border: none;
 }
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --foreground-rgb: 255, 255, 255;
-    --background-start-rgb: 0, 0, 0;
-    --background-end-rgb: 0, 0, 0;
-  }
+.increment button {
```

Comment:
would be better to include the CSS pertaining to the Button component in a file called Button.module.css in the button folder to prevent this file from getting too long